lambdas/data-migration1/tests/test-providers.js

```diff
@@ -295,17 +295,70 @@ test.serial('migrateProviderRecord handles nullable fields on source collection
   );
 });
 
-test.serial('migrateProviderRecord throws RecordAlreadyMigrated error for already migrated record', async (t) => {
+test.serial('migrateProviderRecord throws RecordAlreadyMigrated error if already migrated record is newer', async (t) => {
   const { knex, providerKmsKeyId } = t.context;
-  const fakeProvider = generateFakeProvider();
+  const fakeProvider = generateFakeProvider({
+    updatedAt: Date.now(),
+  });
 
   await migrateProviderRecord(fakeProvider, providerKmsKeyId, knex);
+
+  const olderFakeProvider = {
+    ...fakeProvider,
+    updatedAt: Date.now() - 1000,
+  };
+
   await t.throwsAsync(
-    migrateProviderRecord(fakeProvider, providerKmsKeyId, knex),
+    migrateProviderRecord(olderFakeProvider, providerKmsKeyId, knex),
     { instanceOf: RecordAlreadyMigrated }
   );
 });
 
+test.serial('migrateProviderRecord updates an already migrated record if the updated date is newer', async (t) => {
```

Comment:
```suggestion
test.serial('migrateProviderRecord updates an already migrated record if the updated timestamp on incoming record is newer', async (t) => {
```

Reply:
That's cleaner. Wording on date comparisons is tricky.